benchmark/cmd/querier/main.go

```diff
@@ -194,30 +230,31 @@ func performQuery(logger log.Logger, resultStorage *QueryResult, query string, t
 		// Check prometheus response success code.
 		body, err := ioutil.ReadAll(resp.Body)
 		if err != nil {
-			resultStorage.AddError()
-			level.Error(logger).Log("failed to read query response body")
+			totals.addError()
+			resultStorage.addError()
+			level.Error(logger).Log("err", err)
 			continue
 		}
-		level.Debug(logger).Log("Received query response", body)
 
 		var promResult PromResult
 		if err := json.Unmarshal(body, &promResult); err != nil {
-			resultStorage.AddError()
-			level.Error(logger).Log("failed to unmarshal prometheus result")
+			totals.addError()
```

Comment:
some type of error in those stats would be nice, but maybe those errors does not happen that often, up to you